code/game/shuttle_engines.dm

```diff
@@ -155,6 +155,26 @@
 	bound_height = 96
 	appearance_flags = 0
 
+/obj/structure/shuttle/engine/hugeionengine
+	name = "engine"
+	opacity = 1
+	icon = 'icons/obj/4x7.dmi'
+	icon_state = "huge_ion_engine"
+	desc = "An extremely large bluespace engine used to propel even stations."
+	bound_width = 128
+	bound_height = 224
+	appearance_flags = 0
+
+/obj/structure/shuttle/engine/hugeionafterburn
+	name = "Ion"
+	opacity = 1
+	icon = 'icons/obj/4x7.dmi'
+	icon_state = "huge_ion_afterburn"
+	desc = "Quite hot!."
```

Comment:
What even is this one? name and desc need more detail, same reviews as above apply

Reply:
it's the 'flame' from the engine. I was told to have it as a separate object